app/views/foreman_tasks/tasks/_details.html.erb

```diff
@@ -96,6 +96,18 @@
   <span class="param-value"><%= @task.username %></span>
 </div>
 <div>
+  <span class="param-name"><%= _("Execution type") %>:</span>
+  <span class="param-value"><%= @task.start_at.nil? ? 'Immediate' : 'Delayed' %></span>
+</div>
+<div>
+  <span class="param-name"><%= _("Start at") %>:</span>
+  <span class="param-value"><%= @task.start_at.nil? ? '-' : @task.start_at %></span>
```

Comment:
In the task details page a `-` is used for result so I used it too. I also think it looks cleaner with the dash instead of `N/A`